src/main/java/spaceinvaders/group_22/Game.java

```diff
@@ -288,89 +310,132 @@ public final void setPlayer(final Player newPlayer) {
 		}
 		return spaceBullets;
 	}
+
 	/**
 	 * Gets the explosions currently in this game.
+	 * 
 	 * @return Arraylist of bullets in the game.
 	 */
 	public final ArrayList<Explosion> getExplosions() {
 		return explosions;
 	}
+
 	/**
-	 * Gets the player that is playing this game. 
+	 * Gets the player that is playing this game.
+	 * 
 	 * @return player that is playing this game
 	 */
 	public final Player getPlayer() {
 		return player;
 	}
+
 	/**
 	 * Returns the highscore.
+	 * 
 	 * @return current highscore
 	 */
 	public final int getHighScore() {
 		return highscore;
 	}
+
 	/**
 	 * Gets the canvas width.
+	 * 
 	 * @return width of the canvas.
 	 */
 	public final double getCanvasWidth() {
 		return canvasWidth;
 	}
+
 	/**
 	 * Gets the height of the canvas.
+	 * 
 	 * @return height of the canvas.
 	 */
 	public final double getCanvasHeight() {
 		return canvasHeight;
 	}
+
 	/**
 	 * Gets the alien controller.
+	 * 
 	 * @return the aliencontroller
 	 */
 	public final AlienController getAlienController() {
 		return alienController;
 	}
-		/**
+
+	/**
 	 * Returns the current frame rate.
+	 * 
 	 * @return the current frame rate.
 	 */
 	public final double getTickrate() {
 		return tickrate;
 	}
+
 	/**
-	 * Returns if the player is allowed to shoot at the moment or still in cooldown.
-	 * @return true if the player is allowed to shoot, false if player is in cooldown
+	 * Returns if the player is allowed to shoot at the moment or still in
+	 * cooldown.
+	 * 
+	 * @return true if the player is allowed to shoot, false if player is in
+	 *         cooldown
 	 */
 	public final boolean getShootingAllowed() {
 		return shootingAllowed;
 	}
-	
+
 	/**
 	 * Sets the bullet list.
-	 * @param list ArrayList containing the new bullets
+	 * 
+	 * @param list
+	 *            ArrayList containing the new bullets
 	 */
 	public final void setBullets(final ArrayList<Bullet> list) {
 		bullets = list;
 	}
+
 	/**
-	 * Returns the barricadecontroller in this game, mostly intended for testing purposes...
+	 * Returns the barricadecontroller in this game, mostly intended for testing
```

Comment:
I'm not sure if 'mostly intended for testing' should be included, since a future implementation might use this getter method and then the comment becomes invalid.